simpeg/electromagnetics/time_domain/fields.py

```diff
@@ -273,19 +346,61 @@ def _dhdtDeriv_m(self, tInd, src, v, adjoint=False):
 
 
 class Fields3DElectricField(FieldsTDEM):
-    """Fancy Field Storage for a TDEM simulation."""
-
-    knownFields = {"eSolution": "E"}
-    aliasFields = {
-        "e": ["eSolution", "E", "_e"],
-        "j": ["eSolution", "E", "_j"],
-        "b": ["eSolution", "F", "_b"],
-        # 'h': ['eSolution', 'F', '_h'],
-        "dbdt": ["eSolution", "F", "_dbdt"],
-        "dhdt": ["eSolution", "F", "_dhdt"],
-    }
+    r"""Fields class for storing 3D total electric field solutions.
+
+    This class stores the total electric field solution computed using a
+    :class:`.time_domain.Simulation3DElectricField`
+    simulation object. This class can be used to extract the following quantities:
+
+    * 'e' and 'j' on mesh edges.
+    * 'b', 'dbdt' and 'dhdt' on mesh faces.
+
+    See the example below to learn how fields can be extracted from a
+    ``Fields3DElectricField`` object.
+
+    Parameters
+    ----------
+    simulation : .time_domain.Simulation3DElectricField
+        The TDEM simulation object associated with the fields.
+
+    Example
+    -------
+    The ``Fields3DElectricField`` object stores the total electric field solution
+    on mesh edges. To extract the discrete electric fields and db/dt
+    for all sources and time-steps:
+
+    .. code-block:: python
+
+        f = simulation.fields(m)
+        e = f[:, 'e', :]
+        dbdt = f[:, 'dbdt', :]
+
+    The array ``e`` returned will have shape (`n_edges`, `n_sources`, `n_steps`).
+    And the array ``dbdt`` returned will have shape (`n_faces`, `n_sources`, `n_steps`).
+    We can also extract the fields for a subset of the sources and time-steps as follows:
+
+    .. code-block:: python
+
+        f = simulation.fields(m)
+        e = f[source_list, 'e', t_inds]
+        dbdt = f[source_list, 'dbdt', t_inds]
+
+    """
+
+    def __init__(self, simulation):
+        super().__init__(simulation=simulation)
+        self._knownFields = {"eSolution": "E"}
+        self._aliasFields = {
+            "e": ["eSolution", "E", "_e"],
+            "j": ["eSolution", "E", "_j"],
+            "b": ["eSolution", "F", "_b"],
+            # 'h': ['eSolution', 'F', '_h'],
```

Comment:
Maybe we do the same as we have for _b. That is, it throws a not implemented error. h and b can't be accessed from the e-field solution.

Reply:
Sounds good. Since this PR is docstring-only, I would keep this change to a different PR. I also noticed there are some commented lines in the `_b` method, which should also be removed.